projects/packages/search/src/dashboard/components/record-meter/barchart.jsx

```diff
@@ -0,0 +1,137 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+const Chart = React.lazy( () => import( 'chart.js/auto' ) );
```

Comment:
This was the culprit for me. When I swap the lazy-loading for a direct import, the chart works:

`import Chart from 'chart.js/auto';`

<img width="804" alt="Screen Shot 2022-02-22 at 15 12 30" src="https://user-images.githubusercontent.com/17325/155050731-c1079705-0117-42d1-9758-ce0c5f5a2352.png">

Reply:
Oh, nice find! 👍  thank you! When I was chatting with @kangzj about this, he suggested that I'd need to use lazy load instead. However, perhaps I misunderstood that conversation. Is there any particular reason Jasper (or Chris!) that a direct import is a bad idea here?

Reply:
`chartjs` is pretty big (about 200k after minified, dashboard alone is already 180k+), and it's a dedicated package, so it's a good idea to load it dynamically i.e. lazy loading, which we are using for some libs for Instant Search. However i think this is not a must. As to the issue, I guess, there is some initialization issue when doing lazyloading, which would need some sleuthing. 

Would 400k seem a sensible size? @bluefuton @jsnmoon

Reply:
A quick summary of the situation:

- Jasper recommended we lazy load `chart.js`, which is indeed [nearly 200 kB minified](https://bundlephobia.com/package/chart.js@3.7.1) and dwarfs the dashboard bundle. Lazy loading large chunks of code can be beneficial, especially if we need to prioritize quick time-to-render (e.g. `/` on WordPress.com); if we wanted to ensure that our customers could quickly toggle Search and Instant Search, it'd be a good fit here.

- The current implementation wraps a dynamic `chart.js` import with a `React.lazy` invocation. This pattern is meant to lazily load other React components and does not work with importing general libraries (which we are doing here). If we were, however, we'd also need to wrap the use of such a component with a `Suspense` component as shown in the [docs](https://reactjs.org/docs/code-splitting.html#reactlazy).

To put it simply: lazy loading (aka code splitting) takes a bit more effort to have function correctly in React rendering. Fun fact, [suspense usage is currently discouraged in Preact](https://preactjs.com/guide/v10/switching-to-preact/#suspense-experimental) due to bugs. 🙃 

---

My recommendation here is to import `chart.js` as normal and push toward landing this code into production. Afterward, I recommend we invest time figuring out how to replace our `chart.js` usage with native SVG manipulations. (cc @bluefuton)